src/target/riscv/debug_defines.h

```diff
@@ -3,8 +3,8 @@
  * https://github.com/riscv/riscv-debug-spec/ (40b9a05)
  */
 
-#ifndef DEBUG_DEFINES_H
-#define DEBUG_DEFINES_H
+#ifndef OPENOCD_TARGET_RISCV_DEBUG_DEFINES_H
+#define OPENOCD_TARGET_RISCV_DEBUG_DEFINES_H
```

Comment:
This file is also auto-generated, so I would suggest leaving the header guards as-is here.
https://github.com/riscv-collab/riscv-openocd/blob/f5f5f6dd2aeeccc94f0d61a3eaaa07fd8c379473/src/target/riscv/debug_defines.h#L1-L4
Maybe introduce an option in `riscv-debug-spec` to change the name of the guards?
Regardless of the solution, I believe it should be handled in a separate commit (possibly together with `encoding.h`).

Reply:
Thanks @en-sc - sorry, I didn't realise that that file was also generated elsewhere. I'll undo the change to that file and update the PR a bit later today.

Reply:
Addressed by this commit:

- https://github.com/riscv-collab/riscv-openocd/pull/1103/commits/b874990cc7816ad7faec6c731c9e6a8e3c2ca1d6

Reply:
Sorry - inadvertently used DOS/Windows CR/LF instead of Unix/Linux LF only line endings - fixed with this commit:

- https://github.com/riscv-collab/riscv-openocd/pull/1103/commits/e9cad963b3da9108232cfe7f1571e51ce08812b3